### DISCUSSION TITLE
Github Core Developers got excluded with no rewards, while you pay your internal team members?

### DISCUSSION THREAD

**Discussion by avocadodefi** [zkSync-Community-Hub/zksync-developers#599] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

Hello I am writing to express my concerns again regarding the recent reward distribution by your organization. 

It has come to my attention that none of your core contributors received any rewards, whereas external teams already on your payroll did. In Germany, such actions may be considered "versteckte Gewinnausschüttung" (hidden profit distribution), which could potentially be construed as a legal violation in Berlin. For your reference, here is the link to the GitHub contributors graph: [GitHub Contributors Graph](https://github.com/matter-labs/zksync-era/graphs/contributors).

Furthermore, I would like to address a specific discussion on GitHub regarding this matter: [Closed Topic Discussion](https://github.com/zkSync-Community-Hub/zksync-developers/discussions/576). Your team member, [0xsisyfos](https://github.com/0xsisyfos), provided the following response:

"Hello,

To comply with GDPR rules, we are not providing a public list of eligible GitHub users. Users can check if their profile is eligible on: [Claim Page](https://claim.zknation.io/)."

This response appears to be misleading, as the full list of contributors is publicly available on GitHub, which contradicts the claim that GDPR restricts the publication of such information. Given the serious nature of potential "versteckte Gewinnausschüttung," I believe it would be prudent for your team to consider this matter with greater seriousness.

If necessary, I am willing to support your team in submitting this case to the appropriate German authorities for an internal review. Addressing potential business crimes is not only in the best interest of compliance but also upholds the integrity of your organization's practices.

Thank you for your attention to this matter. I look forward to your prompt response.

Sincerely,

avocadodefi (one of the top 300 web3 coders regarding Electric Crypto Capital Anual Report)

**Comment by 0xsisyfos** [comment, marked as answer]
Hello,

Thank you for your message.

- In general, for contributors you can consult our blog article on the different contributors categories: https://docs.zknation.io/zk-token/zk-airdrop.

- For the specific category "Github Developers", the full list of external project is available here: https://github.com/ZKsync-Association/zknation-data/blob/main/external_project_list.csv, however we are not publishing the list of contributors to these specific projects (we do not own any of these repos).

- If you have specific concerns about our GDPR policy, please consult it here: https://zknation.io/privacy, and contact privacy@zknation.io.
